torchvision/transforms/functional_tensor.py

```diff
@@ -185,17 +185,16 @@ def adjust_hue(img, hue_factor):
     if not (-0.5 <= hue_factor <= 0.5):
         raise ValueError('hue_factor ({}) is not in [-0.5, 0.5].'.format(hue_factor))
 
-    if not _is_tensor_a_torch_image(img):
-        raise TypeError('tensor is not a torch image.')
+    if not (isinstance(img, torch.Tensor) and _is_tensor_a_torch_image(img)):
+        raise TypeError('img should be Tensor image. Got {}'.format(type(img)))
 
     orig_dtype = img.dtype
     if img.dtype == torch.uint8:
         img = img.to(dtype=torch.float32) / 255.0
 
     img = _rgb2hsv(img)
     h, s, v = img.unbind(0)
-    h += hue_factor
-    h = h % 1.0
+    h = torch.fmod(h + hue_factor, 1.0)
```

Comment:
I believe this introduces a change in behavior: [`%` is equivalent to `torch.remainder` and not `torch.fmod`](https://github.com/pytorch/pytorch/blob/4ef12be900e9ec8b81fd45b2f7fe14d648826f64/tools/autograd/templates/python_variable_methods.cpp#L1144). Just to be sure, can you revert back to using `%`?